convert2rhel/initialize.py

```diff
@@ -15,19 +15,20 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-try:
-    from yum import *
+import sys
 
-    # This is added here to prevent a generic try-except in the
-    # `check_package_updates`() function.
-    from yum.Errors import RepoError  # lgtm[py/unused-import]
+from convert2rhel import utils
```

Comment:
I think I would move the `utils.set_locale()` function to this file.  Rationale: We need an initialization that runs before third party libraries are imported.  To do that, we should import as few other things as possible.  convert2rhel.utils currently imports pexpect, rpm, and six and we might add other imports there without realizing that it could break this code.  So moving the setlocale function here seems less fragile to me.

Reply:
Thanks for the suggestions! It actually makes sense to me.